src/providers/compression/compression.ts

```diff
@@ -8,10 +8,10 @@ export class CompressionProvider {
 
   constructor() {}
 
-  extractCatBTestResult(compressedData: string): StandardCarTestCATBSchema {
+  extractCatBTestResult(compressedData: string): CatBUniqueTypes.TestResult {
```

Comment:
can we rename this method to just `extractTestResult`? And then use the Union type